File: meteo.py
#-*- encoding:utf-8 -*-
import requests
from time import strftime,gmtime

dataJSONIles = ('Moutsamoudou','Mamoudzou','Moheli','Moroni')
executionDirectory = "/opt/lampp/htdocs/SiteAeroport/"
print "Date d'éxécution : {0}".format(strftime("%d/%m/%Y -> %H:%M:%S",gmtime()))
for ile in dataJSONIles :
    r = requests.get('http://api.wunderground.com/api/5fe80449e735d30f/conditions/q/COMOROS/'+ile+'.json')
    print "code de renvoi HTTP pour {1} : {0}".format(r.status_code,ile)

    file = None
    if ile == 'Moutsamoudou' :
        file = open(executionDirectory+"Anjouan.json","w")
    elif ile == 'Mamoudzou' :
        file = open(executionDirectory+"Mayotte.json","w")
    else :
        file = open(executionDirectory+ile+".json","w")
        
    file.writelines(r.text)
    file.close()

File: meteo.php
<?php
header( 'Content-Type: application/json' );
ini_set("allow_url_fopen", 1);


$iles = array('Moutsamoudou','Mamoudzou','Moheli','Moroni');

foreach($iles as $ile) {
    $json = file_get_contents('http://api.wunderground.com/api/5fe80449e735d30f/conditions/q/COMOROS/'.$ile.'.json');
    
    switch ($ile) {
    case "Moutsamoudou":
       $fp = fopen('Anjouan.json', 'w');
        break;
    case "Mamoudzou":
        $fp = fopen('Mayotte.json', 'w');
        break;
    default:
       $fp = fopen($ile.'.json', 'w');
        break;
    }
    
    $obj = json_decode($json);
    fwrite($fp, json_encode($obj));
    fclose($fp);
}





?>